website/debug_config.yaml

```diff
@@ -1 +1 @@
-sqlite file: '.storage/debug_flask.db'
+sqlite file: './storage/debug_flask.db'
```

Comment:
I wanted the storage to be hidden, this will make it visible. Is it on purpose?

Reply:
Ho right ok, then we need to update the readme because the . is missing on the mkdir command

Reply:
Or we can just leave it visible as it's just for debug?